manuscript/20.supp-figures.md

```diff
@@ -1,24 +1,24 @@
 ## Supplementary Figures {.page_break_before}
 
-### Performance on an uncensored single lineage of increasing size
```

Comment:
@jclagarde @Farnazmdi you should both go through the supplementary figure captions again. I found quite a lot of basic grammar problems (e.g., articles with plural forms), and the figure titles not describing the figure. I've made some adjustments here, but didn't check them all.

Reply:
Will do by the end of the day. 
@jclagarde could you go first and I will start after you are done.

Reply:
@Farnazmdi Sure thing! I will start after my class ends at 10am

Reply:
@Farnazmdi I finished reading through and editing the supplementary figure captions. Please let me know if there are other things that should be adjusted

Reply:
Thanks, JC. I will take over.

Reply:
@aarmey I made a few changes and I think it should be ready now.